### PR TITLE
Config (setting up model)

### PR DESCRIPTION
**Pull request by sushen25**

### CODE REVIEW

File: alt_evaluator.py [hunk 1]
```diff
@@ -19,6 +19,14 @@ def evaluate(self, model):
 
         with torch.no_grad():
             for _, (images, length_labels, digits_labels, paths) in enumerate(self._loader):
+            length_results = 0
+            digit1_results = 0
+            digit2_results = 0
+            digit3_results = 0
+            digit4_results = 0
+            digit5_results = 0
```
Comment (voirra) [inline alt_evaluator.py:26]: What are the lines above for?

File: convert_train_to_lmdb.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    idx = choose_writer_callback(path_to_lmdb_dirs)
+                    txn = txns[idx]
+
+                    example = example_reader.read_and_convert(digit_struct_mat_file)
+                    if example is None:
+                        break
+
+                    str_id = '{:08}'.format(num_examples[idx] + 1)
+                    txn.put(str_id.encode(), example.SerializeToString())
+                    num_examples[idx] += 1
```
Comment (voirra) [inline convert_train_to_lmdb.py:123]: I made some changes to the original `convert_to_lmdb` around here for getting the image names, if u wanna also include that

Comment (sushen25) [reply convert_train_to_lmdb.py:123]: They're in the other branch `add-validation-losses`, the changes will be saved when we merge

File: alt_evaluator.py (hunk 1 above)
Comment (sushen25) [reply alt_evaluator.py:26]: There are for when there's an error in the lmdb file and the for loop doesn't run, it errors out without these

Comment (voirra) [reply alt_evaluator.py:26]: Oh makes sense, is the `for _, (images, length_labels, digits_labels, paths) in enumerate(self._loader):` line above it extra then?

File: convert_train_to_lmdb.py (hunk 2 above)
Comment (voirra) [reply convert_train_to_lmdb.py:123]: It was in a different file so it won't be applied here.

File: alt_evaluator.py (hunk 1 above)
Comment (sushen25) [reply alt_evaluator.py:26]: oop that line shouldn't be there, will get rid of it

File: convert_train_to_lmdb.py (hunk 2 above)
Comment (sushen25) [reply convert_train_to_lmdb.py:123]: oh right, I'm no longer using this file anyway so just leave it be for now

**Review by voirra — APPROVED** [review]

**Pull request merged**